### ISSUE TITLE
Participants creation: Step 3 does not always load completely 

### ISSUE THREAD

**Issue by MSoeb** [OpenSlides/openslides-client#3849]
**Description:** During participant creation, an error may occur that in step 2 the meeting-related fields are not loaded. Only half of the page is loaded.

How the problem looks like:
![image](https://github.com/user-attachments/assets/2460df74-6db5-4ea7-93e3-7bde1814f768)

**Reproduction:** There is no clear way to reproduce the problem. Just try to create a participant with the following steps.
1. open a meeting
2. Click on the + Icon and create a user.
3. Go to step 3 and check if all content is loaded.

**What should happen:** 
Step 3 in the creation wizzard should always load correctly.

**Comment by reiterl** [comment]
What error do you mean? It works fine here. Step 2 doesn't throw an error and is just optional. Please give a reproduction step to step which is more like "Oh open the wizard".

**Comment by MSoeb** [comment]
The crucial point is, that it doesn't occur on a clear reproduction path. It can happen that the creation step 3 (with personal information and meeting specific information) doesn't load completely. If I could, I would provide a clear reproduction path, but I can't. 

Nevertheless, this is an issue which occurs sometimes and should be at least stay open as - let's say as a reminder - to see that there went something wrong.

**Comment by reiterl** [comment]
Yes, I also have seen it once, see #3985 , I couldn't reproduced it.

**Comment by MSoeb** [comment]
Further information, which could be helpful. I got this error randomly again with error message in console.

![image](https://github.com/user-attachments/assets/c301762c-65e7-483a-b01c-11287e20723b)

Try to figure out, what caused this, but I didn't find any clue. Hopefully, this could help.

**Comment by MSoeb** [comment]
Note: I've added an image how the problem looks like.
